dataprofiler/tests/profilers/test_profile_builder.py

```diff
@@ -729,6 +730,42 @@ def test_correct_total_sample_size_and_counts_and_mutability(self):
         self.assertEqual(col_one_len, len(data['NAME']))
         self.assertEqual(col_two_len, len(data['VALUE']))
 
+    def test_save_and_load(self):
+        datapth = "dataprofiler/tests/data/"
+        test_files = ["csv/guns.csv", "csv/iris.csv"]
+
+        def _clean_report(report):
+            data_stats = report["data_stats"]
+            for key in data_stats:
+                stats = data_stats[key]["statistics"]
+                if "histogram" in stats:
+                    if "bin_counts" in stats["histogram"]:
+                        stats["histogram"]["bin_counts"] = \
+                            stats["histogram"]["bin_counts"].tolist() 
+                    if "bin_edges" in stats["histogram"]:
+                        stats["histogram"]["bin_edges"] = \
+                            stats["histogram"]["bin_edges"].tolist() 
+            return report
+
+        for test_file in test_files:
+            # Create Data and Profiler objects
+            data = dp.Data(os.path.join(datapth, test_file))
+            save_profile = dp.Profiler(data)
+            
+            # Save and Load profile with Mock IO
+            save_profile.save()
+            load_profile = dp.Profiler.load()
```

Comment:
it might be better to test save and load in their own tests

Reply:
Also, what is `load_profile = dp.Profiler.load()` calling load on? I would have assumed it would be the path to the profiler to load, but currently it is empty.

Reply:
It currently has a default path enabled (as does save). Should I remove the default filepath?